.github/workflows/crowdin_sync.yml

```
@@ -18,16 +18,16 @@ jobs:
           npm install -g yarn && yarn install
           git config --global user.name "trezor-ci"
           git config --global user.email "${{ secrets.TREZOR_BOT_EMAIL }}"
-          git checkout -B trezor-ci/crowdin-sync
+          git checkout -B ${{ env.BRANCH_NAME }}
           yarn workspace @trezor/suite translations:download --token=${{ secrets.CROWDIN_PERSONAL_TOKEN }}
           yarn workspace @trezor/suite translations:backport-en
           yarn workspace @trezor/suite translations:format
           yarn workspace @trezor/suite translations:extract
           cat packages/suite-data/files/translations/master.json
           yarn workspace @trezor/suite translations:upload --token=${{ secrets.CROWDIN_PERSONAL_TOKEN }}
-          git add . && git commit -m "chore: crowdin translation update" && git push origin trezor-ci/crowdin-sync -f
+          git add . && git commit -m "chore: crowdin translation update" && git push origin ${{ env.BRANCH_NAME }} -f
           gh config set prompt disabled
           gh pr create --repo trezor/trezor-suite --title "Crowdin translations update" --body "Automatically generated PR for updating crowdin translations." --base develop --label translations
         env:
           GITHUB_TOKEN: ${{ secrets.TREZOR_BOT_TOKEN }}
-          CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
+          BRANCH_NAME: trezor-ci/crowdin-sync-${{ format('{0:yyyy-MM-dd}', github.event.timestamp) }}
```

```suggestion
          BRANCH_NAME: trezor-ci/crowdin-sync-${{ format('{0:yyyy-MM-dd}', github.event.timestamp) }}
```

I am not sure which type of event it is. But if it is `pull_request` then probably we should take time from `created_at`, there is no `timestamp` in any `event` type

---

I think we do not need to format the date, even more, if we do two sync in one day, the problem is there again 😛 I would leave just timestamp there

---

what about now?